src/engine/bank01.asm

```diff
@@ -6112,7 +6112,7 @@ DuelDataToSave: ; 6729 (1:6729)
 ;	dw address, number_of_bytes_to_copy
 	dw wPlayerDuelVariables, wOpponentDuelVariables - wPlayerDuelVariables
 	dw wOpponentDuelVariables, wPlayerDeck - wOpponentDuelVariables
-	dw wPlayerDeck, wc500 + $10 - wPlayerDeck
+	dw wPlayerDeck, wNameBuffer + $10 - wPlayerDeck
```

Comment:
wNameBuffer + $10 here is just wDuelTempList. Probably slipped by in one of my older commits. Leaving this for future reference.